lib/Sanitizer.js

```diff
@@ -10,7 +10,7 @@ var _ = require('lodash');
 class Sanitizer {
 
   bool(value) {
-    return _.isBoolean(value) ? value : vtor.toBoolean(value);
+    return _.isBoolean(value) ? value : vtor.toBoolean((value || '') + '');
```

Comment:
minor, but why the added `+ ''` ? Is that ensure the value gets "cast" to a string?